server/join_test.go

```diff
@@ -53,24 +53,32 @@ func newTestMultiJoinConfig(count int) []*Config {
 
 func waitMembers(svr *Server, c int) error {
 	// maxRetryTime * waitInterval = 5s
-	maxRetryCount := 500
-	waitInterval := 100 * time.Millisecond
+	maxRetryCount := 10
+	waitInterval := 500 * time.Millisecond
+	ctx, cancel := context.WithTimeout(context.Background(), defaultDialTimeout)
+	defer cancel()
+
+	for client := svr.GetClient(); maxRetryCount != 0; maxRetryCount-- {
```

Comment:
Why not directly using `client := svr.GetClient()` above then `for ; maxRetryCount != 0; maxRetryCount--`?